src/modules/mc_pos_control/PositionControl.cpp

```diff
@@ -55,8 +55,25 @@ void PositionControl::updateState(const PositionControlStates &states)
 	_vel_dot = states.acceleration;
 }
 
+void PositionControl::_setCtrlFlagTrue()
```

Comment:
How about `void PositionControl::_setCtrlFlag(bool value)`?

Reply:
true. not sure what I was thinking

Reply:
@MaEtUgR will do the change. thanks